src/services/housing.js

```diff
@@ -31,9 +31,10 @@ import './user'; // Needed for typedef of StudentProfileInfo
  * @typedef ApartmentApplicant
  * @property {Number} ApplicationID Application ID number of this application
  * @property {StudentProfileInfo} Profile The StudentProfileInfo object representing this applicant
- * @property {String} Username The username of this applicant
- * @property {Number} Age The age of the student (in years) (only visible to housing admin)
- * @property {String} Class Class
+ * @property {String} [Username] The username of this applicant
+ * @property {DateTime} [BirthDate] The birthday of this applicant (only visible to housing admin)
+ * @property {Number} [Age] The age of the student (in years) (only visible to housing admin)
+ * @property {String} [Class] Class
```

Comment:
Is there a reason the property names are in brackets? It doesn't seem to match the convention of the rest of the docs.

Reply:
The bracket around variable names is a JSDoc syntax for optional properties. There are other places in 360 where I have seen this used because that is how I first learned that this syntax existed, though I don't remember which file(s)

These are marked as optional because the frontend does not need to include the optional parts when sending the object to the API, (for saving and editing/updating an application), and then ones with private information like the BirthDate, Age, and Probation are also "optional" because they will be blank or null from the API unless the user is authorized to view that information